scripts/run_integration_tests.sh

```diff
@@ -0,0 +1,89 @@
+RAW_CFN_TEMPLATE="cdk.out/ExampleDatadogStack.template.json"
+for ((i=0; i < ${#STACK_CONFIGS[@]}; i++)); do
+    tsc --project tsconfig.json
+    cdk synth --app testlib/integration_tests/stacks/${STACK_CONFIGS[i]} --json --quiet
+
+    # Normalize LambdaVersion IDs
+    perl -p -i -e 's/(LambdaVersion.*")/LambdaVersionXXXX"/g' ${RAW_CFN_TEMPLATE}
+    # Normalize S3Key timestamps
+    perl -p -i -e 's/("S3Key".*)/"S3Key": "serverless\/dd-cdk-construct-integration-test\/dev\/XXXXXXXXXXXXX-XXXX-XX-XXXXX:XX:XX.XXXX\/dd-cdk-construct-integration-test.zip"/g' ${RAW_CFN_TEMPLATE}
+    # Normalize dd_cdk_construct version tag value
+    perl -p -i -e 's/(v\d+.\d+.\d+)/vX.XX.X/g' ${RAW_CFN_TEMPLATE}
+    # Normalize Role names
+    perl -p -i -e 's/("HelloHandlerServiceRole.*")/"HelloHandlerServiceRoleXXXXXXXX"/g' ${RAW_CFN_TEMPLATE}
+     # Normalize Handler names
+    perl -p -i -e 's/("HelloHandler(?!ServiceRole)(\d+|\w+)*")/"HelloHandlerXXXXXXXX"/g' ${RAW_CFN_TEMPLATE}
+    # Normalize AssetParameters
+    perl -p -i -e 's/("AssetParameters.*")/"AssetParametersXXXXXXXXXXXXX"/g' ${RAW_CFN_TEMPLATE}
+    # Normalize CDK Metadata Analytics
+    perl -p -i -e 's/("Analytics": "v.*")/"Analytics": "vX:XXXXXX:XXXXXX"/g' ${RAW_CFN_TEMPLATE}
+    # Normalize S3 Bucket for asset
+    perl -p -i -e 's/(for asset .*")/for asset XXXXXXXXXXXXX"/g' ${RAW_CFN_TEMPLATE}
+    # Normalize Metadata aws:asset:path
+    perl -p -i -e 's/("asset\..*")/"asset.XXXXXXXXXXXXX"/g' ${RAW_CFN_TEMPLATE}
+    # Normalize Datadog Layer Arn versions
+    perl -p -i -e 's/(arn:aws:lambda:sa-east-1:464622532012:layer:Datadog-(Python27|Python36|Python37|Python38|Python39|Node10-x|Node12-x|Node14-x|Extension):\d+)/arn:aws:lambda:sa-east-1:464622532012:layer:Datadog-\2:XXX/g' ${RAW_CFN_TEMPLATE}
+    # Normalize API Gateway timestamps
+    perl -p -i -e 's/("ApiGatewayDeployment.*")/"ApiGatewayDeploymentxxxx"/g' ${RAW_CFN_TEMPLATE}
+
+    cp ${RAW_CFN_TEMPLATE} ${TEST_SNAPSHOTS[i]}
+    echo "===================================="
+    if [ "$UPDATE_SNAPSHOTS" = "true" ]; then
+        echo "Overriding ${CORRECT_SNAPSHOTS[i]}"
+        cp ${TEST_SNAPSHOTS[i]} ${CORRECT_SNAPSHOTS[i]}
+    fi
+
+    echo "Performing diff of ${TEST_SNAPSHOTS[i]} against ${CORRECT_SNAPSHOTS[i]}"
+    set +e # Don't exit right away if there is a diff in snapshots
+    diff ${TEST_SNAPSHOTS[i]} ${CORRECT_SNAPSHOTS[i]}
+    return_code=$?
+    set -e
+    if [ $return_code -eq 0 ]; then
+        echo "SUCCESS: There were no differences between the ${TEST_SNAPSHOTS[i]} and ${CORRECT_SNAPSHOTS[i]}"
+    else
+        echo "FAILURE: There were differences between the ${TEST_SNAPSHOTS[i]} and ${CORRECT_SNAPSHOTS[i]}. Review the diff output above."
+        echo "If you expected the ${TEST_SNAPSHOTS[i]} to be different generate new snapshots by running this command from a development branch on your local repository: 'UPDATE_SNAPSHOTS=true ./scripts/run_integration_tests.sh'"
+        exit 1
```

Comment:
It looks like for each stack, we check for a diff and print a success/failure message. If that snapshot failed, we fail the whole integration test suite and exit.

The problem is that if we have, say, six snapshots, and the first one fails, the entire test suite will exit immediately and we won't run the other five snapshot tests.

Can we make it so that all snapshot tests always run, and then at the end we see a summary of which ones passed and which ones failed? (Similar to how unit tests work)